app/views/mailing_list/steps/_postcode.html.erb

```diff
@@ -1,9 +1,14 @@
-<%= f.govuk_error_summary %>
+<h1>Would you like to hear about teacher training events?</h1>
```

Comment:
I think this needs amending slightly - everyone who signs up to the mailing list gets told about events

The key here is that users are being asked if they want to hear about *local* events by supplying their postcode

The h1 probably needs amending slightly to reflect this
eg '... in your area'